kernel/conversion.ml

```diff
@@ -860,8 +859,7 @@ and convert_return_clause mib mip l2r infos e1 e2 l1 l2 u1 u2 pms1 pms2 p1 p2 cu
       let pms2 = inductive_subst mib u1 pms2 in
       let open Context.Rel.Declaration in
       (* Add the inductive binder *)
-      let dummy = mkProp in
-      let ctx = LocalAssum (Context.anonR, dummy) :: ctx in
+      let ctx = None :: List.map get_value ctx in
```

Comment:
I think you should probably perform the chop + map in a single function.

Reply:
That sounds more complicated than it's worth.

Reply:
Meh, this is suboptimal, although unlikely to matter in practice.